deck/models.py

```diff
@@ -247,3 +247,13 @@ def create_initial_jury(sender, instance, signal, created, **kwargs):
     jury.users.add(instance.author)
     instance.jury = jury
     instance.save()
+
+
+@receiver(post_delete, sender=Proposal)
+def send_proposal_deleted_mail(sender, instance, **kwargs):
+    context = {'event_title': instance.event.title,
+               'proposal_title': instance.title}
+    message = render_to_string('mailing/jury_deleted_proposal.txt', context)
+    subject = _(u'Proposal from %s just got deleted' % instance.event.title)
+    recipients = [jury.email for jury in instance.event.jury.users.all()]
```

Comment:
You can use `instance.event.jury.users.values_list('email', flat=True)` instead of that list comprehension, it will go on the db and grab just the email of the users as a list.


o/